packages/config/config/devices/0x0433/q-light_puck.json

```diff
@@ -20,19 +20,19 @@
             "isLifeline": true
         },
         "2": {
-            "label": "On/Off(S1) / Mirror of endpoint 1, group 2",
+            "label": "On/Off(S1) - Basic command" // Mirror of endpoint 1, group 2,
```

Comment:
You're now missing a comma after the closing `"` (also in the other lines you changed).